dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/Node.java

```diff
@@ -142,6 +145,8 @@ public void setLValue() {
         lvalue = true;
     }
 
+    /** Return field variable inSource. */
```

Comment:
Instead copy and adapt the javadoc from the field and add the missing `@return`.